Makefile

```diff
@@ -21,37 +21,42 @@ build:
 	go build -o build/_output/bin/picchu ./cmd/manager
 
 deps:
-	dep ensure -v
+	go mod tidy
+	go mod vendor
 
-generate: deepcopy defaulter openapi clientset
+generate: deepcopy defaulter openapi clientset crds
```

Comment:
I wasn't autogenerating CRDs because there are certain rules that couldn't be autogenerated properly. They were manually maintained. Be very careful changing them and that kbfd still works with the new rules.